cli/configuration/v1/config.go

```diff
@@ -0,0 +1,46 @@
+package v1
+
+import (
+	"encoding/json"
+
+	"ocm.software/open-component-model/bindings/go/runtime"
+)
+
+var scheme = runtime.NewScheme()
+
+func init() {
+	scheme.MustRegisterWithAlias(&Config{}, runtime.NewUngroupedVersionedType(ConfigType, ConfigTypeV1))
+}
+
+const (
+	ConfigType   = "generic.config.ocm.software"
+	ConfigTypeV1 = "v1"
+)
+
+// Config holds configuration entities loaded through a configuration file.
+// +k8s:deepcopy-gen=true
+type Config struct {
+	Type           runtime.Type    `json:"type"`
+	Configurations []Configuration `json:"configurations"`
+}
+
+// Configuration holds a single configuration entity.
+// +k8s:deepcopy-gen=true
+type Configuration struct {
```

Comment:
Why does thing exist? Couldnt we just use runtime.Raw directly in the array?

Reply:
Actually have this removed locally while refactoring runtime.Type. :D But didn't commit it yet because of our upcoming talk and because my test is failing. :)